src/libraries/System.Text.RegularExpressions/tests/MonoRegexTests.cs

```diff
@@ -70,7 +70,10 @@ public static IEnumerable<object[]> ValidateRegex_MemberData()
 
                     (string Pattern, RegexOptions Options, string Input, string Expected) testCase = allEngineCases[i];
                     yield return new object[] { engine, testCase.Pattern, testCase.Options, results[i], testCase.Input, expected };
-                    yield return new object[] { engine, testCase.Pattern, testCase.Options | RegexOptions.CultureInvariant, results[i], testCase.Input, expected };
+                    if ((testCase.Options & RegexOptions.IgnoreCase) != 0)
```

Comment:
debt here to be cleaned up with the ignore case rationalization later?

Reply:
No, this would remain.  My change here is because we were running every test again with CultureInvariant even if the options didn't include IgnoreCase, and without IgnoreCase, CultureInvariant is a nop.

Reply:
ah yes. there's a couple tests where we randomly pass CultureInvariant without ignore case, but whatever.